meetings/2023-03-22.md

```diff
@@ -63,11 +85,19 @@ URL: https://github.com/solid/specification/issues/161
 URL: https://github.com/solid/notifications/issues/157
 
 * SC: Move work to solid/specification?
-
+* eP: Question to PAC about latest work on deltas in RDF, maybe the cannonicalization wg
+* TBL: rdflib can handle deltas already, data doesn't have an ocean of blank nodes, no need to panic about blank nodes and diffs. If we have a format which client sends a patch to the server, the same format can be interpreted by the client if received in the notification.
+* PAC: I totally agree with Tim, blank nodes are not so bad in practice. The cannonicalization algorithm could be a one way to use in patch. Again as Tim said it could be an overkill, in most cases WHERE clauses are easier to identify those blank nodes.
+* TBL: Maybe the metadata that is added to notification, the changlog is not only the changes but also the basic stuff you need to undo, blame, basically offline first.
+* eP: Authorization system would need to take it into account if we disclose who made the change to the resource.
+* TBL: concept of who is the creator of the document is impmortant
 
 ---
 
-### Topic
-URL: 
+### Solid WG
+URL: https://github.com/solid/solid-wg-charter
+
+* Proposed by PAC
+* PAC: The work is progressing, soon we are going to submit proposal to horizontal review group. When this is done it will go to members for the approval. Don't hasitate to have another look at the charter.
+* PAC: I would like to setup pool which was done for previous charter. It was called expression of support / expression of interest. People and origanizations could record interest in the future working group.
```

Comment:
```suggestion
* PAC: The work is progressing; soon we will submit the proposal to horizontal review. When this is done, it will go to members for their approval. Don't hesitate to have another look at the charter.
* PAC: I would like to set up a pool, as was done for previous charter. It was called "expression of support or interest". People and organizations could record interest in the future working group.
```